charts/memgraph/values.yaml

```diff
@@ -79,6 +79,12 @@ persistentVolumeClaim:
   userStorageAccessMode: "ReadWriteOnce"
   userMountPath:
 
+  ## Create a Persistant Volume Claim for core dumps.
+  createCoreDumpsClaim: true
```

Comment:
Don't forget to change the default,  since I guess you probably do not want this PVC by default.